src/methods.jl

```diff
@@ -194,12 +216,14 @@ trim(frame, idxs) = copy(trimview(frame, idxs))
 
 
 """
-    trimview(frame::AbstractArray, idxs)
+    trimview(frame, idxs)
 
 Trims the `frame` to remove the region specified by idxs.
 
 This function is same as the [`trim`](@ref) function but returns a view of the frame.
 
+`frame` cannot be a string. Function output inherits the type of first parameter and header file of output (if applicable) is same as of the first parameter.
```

Comment:
combo of both my previous comments! The first half makes me ask "why" and the second half is a little hard to read/hard to understand.